lib/RouteStore.js

```diff
@@ -23,6 +25,7 @@ var RouteStore = createStore({
         this.currentUrl = null;
         this.currentRoute = null;
         this.currentNavigate = null;
+        this.currentNavigateError = null;
```

Comment:
I have concerns about leaking a server side stack trace that could contain sensitive info.